easybuild/easyblocks/s/superlu.py

```diff
@@ -88,11 +98,24 @@ def configure_step(self):
             self.cfg.update('configopts', '-DBLA_VENDOR="%s"' % toolchain_blas)
 
         elif toolchain_blas == 'OpenBLAS':
-            # Unfortunately, OpenBLAS is not recognized by FindBLAS from CMake,
-            # we have to specify the OpenBLAS library manually
-            openblas_lib = os.path.join(get_software_root('OpenBLAS'), get_software_libdir('OpenBLAS'), "libopenblas.a")
-            self.cfg.update('configopts', '-DBLAS_LIBRARIES="%s;-pthread"' % openblas_lib)
-
+            if LooseVersion(cmake_version) >= LooseVersion('3.6'):
+                self.cfg.update('configopts', '-DBLA_VENDOR="%s"' % toolchain_blas)
+            else:
+                # Unfortunately, OpenBLAS is not recognized by FindBLAS from CMake,
+                # we have to specify the OpenBLAS library manually
+                openblas_lib = os.path.join(get_software_root('OpenBLAS'), get_software_libdir('OpenBLAS'),
+                                            "libopenblas.a")
+                self.cfg.update('configopts', '-DBLAS_LIBRARIES="%s;pthread"' % openblas_lib)
+
+        elif toolchain_blas == 'FlexiBLAS':
+            if LooseVersion(cmake_version) >= LooseVersion('3.19'):
+                self.cfg.update('configopts', '-DBLA_VENDOR="%s"' % toolchain_blas)
+            else:
+                # Unfortunately, FlexiBLAS is not recognized by FindBLAS from CMake,
+                # we have to specify the FlexiBLAS library manually
+                flexiblas_lib = os.path.join(get_software_root('FlexiBLAS'), get_software_libdir('FlexiBLAS'),
+                                             "libflexiblas.so")
+                self.cfg.update('configopts', '-DBLAS_LIBRARIES="%s;pthread"' % flexiblas_lib)
```

Comment:
```suggestion
                self.cfg.update('configopts', '-DBLAS_LIBRARIES="%s;pthread"' % flexiblas_lib)

```
...but does help here